dapp/src/scss/components/home/homepage.scss

```diff
@@ -65,9 +65,18 @@
     display: flex;
     align-items: baseline;
     margin: auto 0;
+    @include element(dl){
+      display: none;
+    }
 
     @include breakpoint(small down) {
       margin: auto;
+      @include element(dl){
+        display: block;
+      }
+      @include element(default){
```

Comment:
@NikolayRn What is `default`?

Reply:
Its the name of old home image element, not the best naming...

<img width="470" alt="Screen Shot 2022-06-16 at 22 10 57" src="https://user-images.githubusercontent.com/68244817/174147265-20252a3a-c14a-4b17-99e7-8f214d991299.png">